addons/base/tests/models.py

```diff
@@ -34,7 +34,6 @@ def ExternalAccountFactory(self):
         pass
 
 
-@pytest.mark.enable_quickfiles_creation
```

Comment:
`@pytest.mark.enable_quickfiles_creation` slows down tests, so I've removed it where it's no longer needed. Most of these `enable_quickfiles_creation` marks aren't necessary anymore because we no long need to exclude QuickfilesNodes.